services/data-info/src/data_info/util/irods.clj

```diff
@@ -11,8 +11,19 @@
             [data-info.util.config :as cfg])
   (:import [clojure.lang IPersistentMap]
            [java.util UUID]
+           [java.io IOException]
+           [org.irods.jargon.core.exception JargonException]
            [org.apache.tika Tika]))
 
+(defmacro catch-jargon-io-exceptions
```

Comment:
(here's where you're looking for the implementation of the new macro, if you're having trouble digging it out of the whitespace changes)